.github/workflows/design-workflow.yml

```diff
@@ -0,0 +1,60 @@
+name: Design Changes Workflow
+
+on:
+  workflow_dispatch:
+    inputs:
+      description:
+        description: 'What are we updating (eg. update button destructive color)?'
+        default: ''
+        required: true
+      jira:
+        description: 'What is the related JIRA (eg. SW-1234)?'
```

Comment:
Total nitpick: it's "Jira" nowadays (they changed from the all-caps name a few years back).